### DISCUSSION TITLE
第5节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#330] [笔记]
第5节课笔记(3班)

**Comment by 443752009** [comment]
http://t.csdnimg.cn/jzv7C

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/12/mmtutorial-5-note/

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/bc4lg0nqlzm2zn3o?singleDoc# 《LMDeploy 大模型量化部署实践笔记》

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/KEths

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/677885598

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]LMDeploy 大模型量化部署实践 - 知乎 (zhihu.com)](https://zhuanlan.zhihu.com/p/678685048)

**Comment by Sampson425** [comment]
https://blog.csdn.net/m0_49908454/article/details/135581858

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/679029455

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135853852
